projects/configs/tag-operations/mri.yaml

```diff
@@ -110,6 +79,9 @@
   group: 0x0008
   element: 0x9209
   op: "keep"
+#################################### 0010 Group ###################################
+#
+#
 - name: "Patients Name"
   group: 0x0010
   element: 0x0010
```

Comment:
We shouldn't be keeping the patients name, instead we should be replacing it, and also I think this is in base? so should be removed here?

Reply:
[0b63011](https://github.com/SAFEHR-data/PIXL/pull/498/commits/0b63011e0e8e6bce28b7e64ccba7f9e733bff945)